src/test/common/Throttle.cc

```diff
@@ -217,41 +218,36 @@ TEST_F(ThrottleTest, wait) {
 }
 
 TEST_F(ThrottleTest, destructor) {
-  Thread_get *t;
-  {
-    int64_t throttle_max = 10;
-    Throttle *throttle = new Throttle(g_ceph_context, "throttle", throttle_max);
+  int64_t throttle_max = 10;
+  Throttle *throttle = new Throttle(g_ceph_context, "throttle", throttle_max);
```

Comment:
I had thought of that, but figured I wouldn't since I was just moving those allocations around. I'll be happy to use unique_ptr there.